clinicaltrials.gov.js

```diff
@@ -0,0 +1,273 @@
+{
+	"translatorID": "874d70a0-6b95-4391-a681-c56dabaa1411",
+	"label": "clinicaltrials.gov",
+	"creator": "Ryan Velazquez",
+	"target": "^https://(www\\.)?clinicaltrials\\.gov/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2020-04-01 14:00:00"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2020 Ryan Velazquez
+	
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function detectWeb(doc, url) {
+	if (url.includes("https://clinicaltrials.gov/ct2/results")) {
```

Comment:
can we not make multiples work or have you just not gotten to it?
We strongly prefer having multiples available -- especially for things like systematic reviews this would seems super useful.
In any case, we should then just exclude them via target regex - and make the target regex as well as `detectWeb()` more restrictive overall: in its current form the translator would detect e.g. the homepage and help pages as reports and then fail on import. This should never happen.

Reply:
I made the target more restrictive. I wasn't sure if any additional checks should go in `detectWeb()`; it seemed like the target was the preferred place to make it more restrictive.

I thought the single case was more important than the multiples and wanted to focus my limited time on getting that done first. I know we need the single use case for the active https://github.com/greenelab/covid19-review and wasn't aware of a use case for the multiples (I assume this would be getting all the citations for what the clinicalTrials website returns from a search?). I'd be willing to help get that incorporated but I'm not sure how much time I'll have in the next few weeks.